### PR TITLE
Bert pipe

### PR DESCRIPTION
**Pull request by SoluMilken**
Topic: Use an Uttpipe to imitate a BERT Tokenizer.
BERT has three tokenizers - Basic Tokenizer, WordPiece Tokenizer, FullTokenizer.

Bert Tokenizer <==> Uttpipe
BasicTokenizer with lowercase = True   <==>  basic_pipe in `uttut/pipeline/bert/basic.py`
WordPieceTokenizer  <==>  word_piece_pipe in `uttut/pipeline/bert/word_piece.py`
FullTokenizer  <==> full_pipe in `uttut/pipeline/bert/full.py`

### CODE REVIEW

File: uttut/pipeline/bert/tests/test_basic.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from .tokenization import BasicTokenizer
```
Comment (absolutelyNoWarranty) [inline uttut/pipeline/bert/tests/test_basic.py]:
```suggestion
from .tokenization import BasicTokenizer as BertBasicTokenizer
```

File: uttut/pipeline/bert/tests/test_word_piece.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from .tokenization import WordpieceTokenizer
```
Comment (absolutelyNoWarranty) [inline uttut/pipeline/bert/tests/test_word_piece.py]:
```suggestion
from .tokenization import WordpieceTokenizer as BertWordpiecTokenizer
```

Comment (SoluMilken) [reply uttut/pipeline/bert/tests/test_word_piece.py]: 003a606

File: uttut/pipeline/bert/tests/test_basic.py (hunk 1 above)
Comment (SoluMilken) [reply uttut/pipeline/bert/tests/test_basic.py]: 003a606

**Pull request merged**